README.md

```diff
@@ -17,19 +17,22 @@
 If you're reading this, you're as interested as we are in promoting Free and Open Source Software (FOSS) and the
 ever growing Web3 ecosystem.
 
-This page will cover the basics of the Radicle Grants Program's ethos, application processes, and structure.
+This page will cover the basics of the Radworks Grants Program's ethos, application processes, and structure.
 
 ### Guidelines
 
-The focus is on technical projects, such as additions to core Radicle software or components of Web3 infrastructure 
-that interact with Radicle, or adoption of some part of the [Radicle stack](https://github.com/radicle-dev) (e.g. [Radicle Drips SDK](https://github.com/radicle-dev/drips-js-sdk)).
+The focus is on technical projects that support the developmet of the Radicle & Drips networks. This include: 
+- Additions to core Radicle software or components of Web3 infrastructure 
+that interact with Radicle
+- [Insert Drips note]
```

Comment:
Additions to core Drips software or components, or compelling adoption of tooling, such as [Radicle Drips SDK](https://github.com/radicle-dev/drips-js-sdk)